mne/connectivity/spectral.py

```diff
@@ -613,8 +647,9 @@ def spectral_connectivity(data, method='coh', indices=None, sfreq=2 * np.pi,
         number of time points as stc_*. The array-like object can also
         be a list/generator of array, shape =(n_signals, n_times),
         or a list/generator of SourceEstimate or VolSourceEstimate objects.
-    method : string | list of string
-        Connectivity measure(s) to compute.
+    method : string | list of string | callable
```

Comment:
I would say let's support internally / privately `callable` but not tell users about it. We don't want to have to explain it / support it if we don't have to.